src/swim/swim_internal.h

```diff
@@ -80,7 +80,7 @@ extern "C" {
 #endif
 
 /** SWIM protocol parameter defaults */
-#define SWIM_PROTOCOL_PERIOD_LEN 2000	/* milliseconds, should NOT be less
+#define SWIM_PROTOCOL_PERIOD_LEN 2400	/* milliseconds, should NOT be less
```

Comment:
we should add static assert here to enforce the restriction that you mention in comments

Reply:
This is not critical. This is just recommendation.